src/scripts/background.js

```diff
@@ -181,7 +181,11 @@ window.TogglButton = {
             if (xhr.status === 200) {
               browser.tabs.query({ active: true, currentWindow: true })
                 .then(filterTabs(function (tabs) {
-                  browser.tabs.sendMessage(tabs[0].id, { type: 'sync' });
+                  browser.tabs.sendMessage(tabs[0].id, { type: 'sync' }).catch(err => {
+                    if (err.message !== 'Could not establish connection. Receiving end does not exist.') {
```

Comment:
It's so weird... They have no stacktrace, no errorcode... Maybe because they're part of the extension API they're not properly filled in?
I also found no documentation about possible errors/error codes from these methods... So... This is the best I got. Seemed at least consistent across browsers